tests/libtest.sh

```diff
@@ -123,9 +123,7 @@ files_are_hardlinked() {
 }
```

Comment:
Would be good to add a check in `files_are_hardlinked()` that `f1` & `f2` are non-empty. Otherwise, we'll succeed even if the files don't exist at all.